docs/static/troubleshoot/ts-azure.asciidoc

```diff
@@ -0,0 +1,81 @@
+[[ts-azure]]
+==== Azure Event Hub issues and solutions
+ 
+[discrete] 
+[[ts-azure-http]]
+===== Event Hub plugin can't connect to Storage blob (input)
+
+*Symptoms* 
+
+Azure EventHub plugin can't connect to the blob storage:
+
+-----
+[2024-01-01T13:13:13,123][ERROR][com.microsoft.azure.eventprocessorhost.AzureStorageCheckpointLeaseManager][azure_eventhub_pipeline][eh_input_plugin] host logstash-a0a00a00-0aa0-0000-aaaa-0a00a0a0aaaa: Failure while creating lease store
+com.microsoft.azure.storage.StorageException: The client could not finish the operation within specified maximum execution timeout.
+-----
+
+The plugin can't complete the registration phase because can't connect to the Azure Blob Storage configured
+in the plugin `storage_connection` setting.
+
+
+*Background*
+
+Azure Event Hub plugin can share the offset position of a consumer group with
+other consumers only if configures the Blob Storage connection setting.
+Despite EventHub uses the AMQP protocol to transfer data, the Blob Storage uses a
+library which leverages the JDK's http client, `HttpURLConnection`.
+To troubleshoot HTTP connection problems, maybe related to proxy settings, the
+logging level for this part of the JDK has to be increased. The problem is that
+JDK uses Java Util Logging for its internal logging needs, which is not configurable
+with the standard `log4j2.properties` shipped with {ls}.
+
+*Possible solutions*
+
+* Configure {ls} settings to enable the JDK logging.
+
+*Details*
+
+To enable JDK logging on {ls}, needs to:
+
+* Create a properties file with the logging definitions for Java Util Logging (JUL).
+* Configure a JVM property to inform JUL to use such definitions file.
+
+*JUL definitions*
+
+Create a file (for example `<LS_HOME>/conf/jul.properties`) to define logging levels, handlers and loggers:
+
+[source,txt]
+-----
+handlers= java.util.logging.ConsoleHandler,java.util.logging.FileHandler
+.level= ALL
+java.util.logging.FileHandler.pattern = <USER's LOGS FOLDER>/logs/jul_http%u.log
+java.util.logging.FileHandler.limit = 50000
+java.util.logging.FileHandler.count = 1
+java.util.logging.FileHandler.level=ALL
+java.util.logging.FileHandler.maxLocks = 100
+java.util.logging.FileHandler.formatter = java.util.logging.SimpleFormatter
+
+java.util.logging.ConsoleHandler.level = INFO # or put FINE
+java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
+
+# defines the logger we are interested in
+sun.net.www.protocol.http.HttpURLConnection.level = ALL
+-----
+
+With this configuration the log file will be created in a path chosen by the user (`<USER's LOGS FOLDER>/logs/`) and
+the only logger interested is `sun.net.www.protocol.http.HttpURLConnection`, at every level,
```

Comment:
the logger of interest is `sun.net.www.protocol.http.HttpURLConnection`, at every level,